tgt_grease/core/Configuration.py

```diff
@@ -173,6 +173,9 @@ def DefaultConfig():
                     'enabled': False,
                     'token': None,
                     'room': None
+                },
+                'Slack': {
```

Comment:
Add an `enabled` property and default it to `False` that way the notification engine will know whether to send messages or not